### PR TITLE
ryan-koshiol adds Saint Cloud file

### LINKED ISSUE #10: ryan-koshiol practices using GitHub Flow
- [x] Create a branch of of master
- [ ] Add some commits
- [ ] Open a pull request
- [x] Collaborate
- [x] Merge when we get at least 1 approval 👍

### PR DESCRIPTION
**Pull request by ryan-koshiol**
The purpose of this PR is to add Saint Cloud file to the hometown files.

This PR closes #10

### CODE REVIEW

File: ryan-koshiol-SaintCloud.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+- Val's hamburgers
+
+### Places
+- Quary Park
+- The Press
+ -SCSU  
+
+### Sports
+- Rox
+- Riverside Park - disc golf
```
Comment (jerry-ennett) [inline ryan-koshiol-SaintCloud.md]: Do you like disc golf?

**Comment by ryan-koshiol** [comment]
@jerry-ennett wasn't anything actionable in your comment, but I updated some stuff anyway c555713

File: ryan-koshiol-SaintCloud.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+### Places
+- Quary Park
+  - Swim in a granite Quary
+  - Groomed and lighted ski trails
+- The Press
+- SCSU  
+
+### Sports
+- Rox
+- Riverside Park - disc golf (Championship level course)
```
Comment (jerry-ennett) [inline ryan-koshiol-SaintCloud.md:15]: You should add one more park.

**Review by jerry-ennett — APPROVED** [review]
This is very nicely done!

**Pull request merged**